language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/providers/context/ImportOrgNameNodeContext.java

```diff
@@ -103,4 +103,16 @@ private static LSCompletionItem getImportCompletion(CompletionContext context, S
 
         return new StaticCompletionItem(context, item, StaticCompletionItem.Kind.MODULE);
     }
+
+    @Override
+    public void sort(CompletionContext context, ImportOrgNameNode node, List<LSCompletionItem> completionItems) {
+        String orgName = node.orgName().text();
+        if (!"ballerina".equals(orgName)) {
```

Comment:
Would be better to move this to a constant

Reply:
+1. Sorting API is finalising and yes, all such check will be done against to a set of constants.